src/nav/navActions.js

```diff
@@ -13,11 +13,35 @@ import { getSameRoutesCount } from '../selectors';
 
 export const navigateBack = (): NavigationAction => StackActions.pop({ n: getSameRoutesCount() });
 
-// Other stack routes reached through `navReducer`:
-//    StackActions.push({ routeName: 'loading' });
-//    StackActions.push({ routeName: 'realm' });
-//    StackActions.push({ routeName: 'account' });
-//    StackActions.push({ routeName: 'main' });
+/**
+ * "Reset" actions, to explicitly prohibit back navigation.
+ */
```

Comment:
nit: plain non-jsdoc comment, because this doesn't attach to a particular single item that comes just after it.

(As is, this will appear when hovering on a call site of `resetToLoading` in an IDE, because that happens to be the first thing after this.)